.github/workflows/stale-issues-pr.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     steps:
       - uses: actions/stale@v6
         with:
+          operations-per-run: 250
```

Comment:
By reading the docs it seems that this option is to protect requests to Github API to be rate limited:

> GitHub has a [rate limit](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) and if reached will block these API calls for one hour 

The default value is 30 and I am afraid that increasing it to 250 would cause us to be rate limited.
WDYT?

Reply:
I did some poking around we should have 1000 hits per hour and this runs only once a day in the early hours of the night.

Reply:
Also the 250 is operations which means that it would at most eat 250 api calls for that run, but it will actually end up also being way less. That would be a worse case when needing to set status on 250 issues/pr's